spec/dummy/config/initializers/new_framework_defaults.rb

```diff
@@ -18,7 +18,7 @@
 Rails.application.config.active_record.belongs_to_required_by_default = true
 
 # Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = false
+# ActiveSupport.halt_callback_chains_on_return_false = false
```

Comment:
Would you mind removing this section instead of commenting it out? Also, could you explain a little about the reason for removing it? It looks like it has been depreciated but do you think this will affect anything? What was the reason for it before?